VERSION

```diff
@@ -1 +1 @@
-2.1.26
+2.1.25
```

Comment:
Why did you revert this change?

IIRC, we can't do internal build via Jenkins if the VERSION file is not set to this value.